ui/src/components/PortfolioForm.tsx

```diff
@@ -141,57 +134,57 @@ export default function PortfolioForm() {
           </h3>
 
           <div className="setting-row">
-            <label>Stake when tokens arrive:</label>
-            <input
-              type="checkbox"
-              checked={settings.stake || false}
-              onChange={() => toggleSetting('stake')}
-            />
+            <label>Polling Frequency:</label>
           </div>
           <div className="button-group">
             <button
```

Comment:
Since this is a choice of exclusive options, I **suggest** using radio buttons.

and `onChange` rather than `onClick`?